### PR TITLE
Rename to `brew bundle`.

### PR DESCRIPTION
**Pull request by mikemcquaid**
I've heard a few complaints about the name and I can never remember the actual command name(s). After renaming keep the old naming around as symlinked aliases so they can still be called.

### CODE REVIEW

**Comment by andrew** [comment]
I'll be sad to see the name go but it makes sense for it to be more predictable :+1:

**Comment by xu-cheng** [comment]
So we will change the repo name as well?

File: cmd/brew-bundle.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  brew bundle cleanup   uninstall all Homebrew formulae not listed in Brewfile
+
+  Options:
+  -v, --verbose          print verbose output
+  --force                force overwrite existed Brewfile
+  --dry-run              list formulae rather than actual uninstalling them
+  --file=<path>          set Brewfile path
+  --global               set Brewfile path to $HOME/.Brewfile
+  -h, --help             show this help message and exit
+  --version              show the version of bundle
```
Comment (xu-cheng) [inline cmd/brew-bundle.rb]: Maybe want the right part to be aligned with `Usage` part.

**Comment by xu-cheng** [comment]
:+1:

**Comment by mikemcquaid** [comment]
Renamed repo and merged. Thanks for the understanding @andrew :heart:.

**Pull request merged**